src/index.html

```diff
@@ -11,7 +11,168 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+            Surname:
+          <input 
+           type="text" 
+           name="surname"
+           autocomplete="off"
+          >
+        </label>
+      
+        <label class="form-field">
+            Name:
+          <input 
+           type="text" 
+           name="Name"
+          >
+        </label>
+        <label class="form-field">
+            How old are You?
+          <input 
+           type="number" 
+           name="old"
+           min="1"
+           max="100"
+           value="12"
+          >
+        </label>
+    
+        <label class="form-field">
+            Full date of birth:
+          <input 
+           type="date" 
+           name="Full date"
+          >
```

Comment:
```suggestion
          <input type="date" name="Full date">
```
if there is less then 3 attributes, there is no need in such code style